src/css/support.js

```diff
@@ -53,6 +53,11 @@ define( [
 		div = null;
 	}
 
+	// Rounding up the exact checking match values
+	function roundPixelMeasures( measure ) {
+		return Math.round( parseFloat( measure ) ) + "px";
```

Comment:
Adding "px" seems pointless, we should just return and compare against the numeric value (saving a few bytes).

Reply:
Removed "px" and used the numeric values to compare, since it saves a few bytes in both the places as suggested by you.